python/az/aro/azext_aro/tests/latest/unit/test_validators.py

```diff
@@ -128,47 +132,59 @@ def test_validate_client_id(test_description, namespace, expected_exception):
 
 test_validate_client_secret_data = [
     (
-        "should not raise any exception when isCreate is false and enable_managed_identity is None",
-        False,
-        Mock(client_id=None),
+        "should not raise any exception when namespace.client_secret is None",
+        True,
+        Mock(client_secret=None),
         None
     ),
     (
-        "should raise MutuallyExclusiveArgumentError when isCreate is false and enable_managed_identity is True",
-        False,
+        "should raise MutuallyExclusiveArgumentError when enable_managed_identity is True",
+        True,
         Mock(client_secret="123", enable_managed_identity=True),
-        None
+        MutuallyExclusiveArgumentError
     ),
     (
-        "should not raise any exception when namespace.client_secret is None",
+        "should raise MutuallyExclusiveArgumentError when isCreate is true and platform_workload_identities is present",
         True,
-        Mock(client_secret=None),
-        None
+        Mock(client_secret="123", platform_workload_identities=[Mock(resource_id='Foo')]),
+        MutuallyExclusiveArgumentError
+    ),
+    (
+        "should raise MutuallyExclusiveArgumentError when isCreate is false and platform_workload_identities is present",
+        False,
+        Mock(client_secret="123", platform_workload_identities=[Mock(resource_id='Foo')]),
+        MutuallyExclusiveArgumentError
     ),
     (
-        "should raise RequiredArgumentMissingError exception when namespace.client_id is None and client_secret is not None",
+        "should raise RequiredArgumentMissingError exception when isCreate is true, namespace.client_id is None, and client_secret is not None",  # pylint: disable=line-too-long
         True,
-        Mock(client_id=None, client_secret="123"),
+        Mock(client_id=None, client_secret="123", platform_workload_identities=None),
         RequiredArgumentMissingError
     ),
     (
-        "should raise RequiredArgumentMissingError exception when can not crate a string representation from namespace.client_id because it is empty",
+        "should raise RequiredArgumentMissingError exception when isCreate is true and can not create a string representation from namespace.client_id because it is empty",  # pylint: disable=line-too-long
         True,
-        Mock(client_id="", client_secret="123"),
+        Mock(client_id="", client_secret="123", platform_workload_identities=None),
         RequiredArgumentMissingError
     ),
     (
-        "should raise RequiredArgumentMissingError exception when can not crate a string representation from namespace.client_id because it is None",
+        "should raise RequiredArgumentMissingError exception when isCreate is true and can not create a string representation from namespace.client_id because it is None",  # pylint: disable=line-too-long
```

Comment:
This looks like a duplicate of the test case on line 159.

Reply:
yup, I'll get rid of it.